Fix for pyolog and opyhd

stuwilkins opened the pull request:
Fixes for ophyd and pyolog. Sorry @tacaswell, @dchabot I forgot to put this PR in in my reduced state yesterday.

tacaswell commented:
I have modified recipes also not checked in on one of the srx machines, let
me reconcile those before this gets merged please.

On Thu, Feb 12, 2015, 08:07 Stuart Wilkins <notifications@github.com> wrote:

> Fixes for ophyd and pyolog. Sorry @tacaswell
> <https://github.com/tacaswell>, @dchabot <https://github.com/dchabot> I
> forgot to put this PR in in my reduced state yesterday.
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/NSLS-II/conda-prescriptions/pull/16
> Commit Summary
>
>    - ENH : Added pyolog
>    - BUG : no ipython in ophyd
>
> File Changes
>
>    - *M* ophyd/meta.yaml
>    <https://github.com/NSLS-II/conda-prescriptions/pull/16/files#diff-0>
>    (1)
>    - *A* pyolog/build.sh
>    <https://github.com/NSLS-II/conda-prescriptions/pull/16/files#diff-1>
>    (4)
>    - *A* pyolog/meta.yaml
>    <https://github.com/NSLS-II/conda-prescriptions/pull/16/files#diff-2>
>    (33)
>
> Patch Links:
>
>    - https://github.com/NSLS-II/conda-prescriptions/pull/16.patch
>    - https://github.com/NSLS-II/conda-prescriptions/pull/16.diff
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NSLS-II/conda-prescriptions/pull/16>.
>

stuwilkins commented:
No problem

Best wishes,
Stuart


On Feb 12, 2015, at 08:45, Thomas A Caswell <notifications@github.com<mailto:notifications@github.com>> wrote:

I have modified recipes also not checked in on one of the srx machines, let
me reconcile those before this gets merged please.

On Thu, Feb 12, 2015, 08:07 Stuart Wilkins <notifications@github.com<mailto:notifications@github.com>> wrote:

> Fixes for ophyd and pyolog. Sorry @tacaswell
> <https://github.com/tacaswell>, @dchabot <https://github.com/dchabot> I
> forgot to put this PR in in my reduced state yesterday.
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
> https://github.com/NSLS-II/conda-prescriptions/pull/16
> Commit Summary
>
> - ENH : Added pyolog
> - BUG : no ipython in ophyd
>
> File Changes
>
> - *M* ophyd/meta.yaml
> <https://github.com/NSLS-II/conda-prescriptions/pull/16/files#diff-0>
> (1)
> - *A* pyolog/build.sh
> <https://github.com/NSLS-II/conda-prescriptions/pull/16/files#diff-1>
> (4)
> - *A* pyolog/meta.yaml
> <https://github.com/NSLS-II/conda-prescriptions/pull/16/files#diff-2>
> (33)
>
> Patch Links:
>
> - https://github.com/NSLS-II/conda-prescriptions/pull/16.patch
> - https://github.com/NSLS-II/conda-prescriptions/pull/16.diff
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NSLS-II/conda-prescriptions/pull/16>.
>

—
Reply to this email directly or view it on GitHub<https://github.com/NSLS-II/conda-prescriptions/pull/16#issuecomment-74072067>.

tacaswell commented on pyolog/meta.yaml:
```diff
@@ -0,0 +1,33 @@
+
+package:
+    name: pyolog
+    version: "0.3.0"
```
The version of pyolog scraped from the tags violently disagrees.
